cron/src/requirements.txt

```diff
@@ -0,0 +1,24 @@
+atomicwrites==1.3.0
+attrs==19.1.0
+colorama==0.4.1
+entrypoints==0.3
+flake8==3.7.7
+freezegun==0.3.12
+importlib-metadata==0.18
+mccabe==0.6.1
+more-itertools==7.0.0
+packaging==19.0
+pluggy==0.12.0
+psycopg2==2.8.3
+py==1.8.0
+pycodestyle==2.5.0
+pyflakes==2.1.1
+pyparsing==2.4.0
+pytest==4.6.3
+python-dateutil==2.8.0
+python-dotenv==0.10.3
+six==1.12.0
+SQLAlchemy==1.3.5
+SQLAlchemy-Utils==0.34.0
+wcwidth==0.1.7
+zipp==0.5.1
```

Comment:
Majority of those are not needed, if I understand correctly only these three are needed:

freezegun==0.3.12
pytest==4.6.3
SQLAlchemy==1.3.5

Reply:
Requirements files are typically used to hold the result of `pip freeze`. `pip freeze` on the other hand outputs all installed packages, so as a result requirements file contains all packages, not only the top ones.
More [here](https://pip.pypa.io/en/stable/user_guide/#requirements-files).